test/test-files/appsync.ts

```diff
@@ -0,0 +1,66 @@
+import * as appsync from "@aws-cdk/aws-appsync-alpha";
+import { App, Stack } from "aws-cdk-lib";
+import { AppsyncResolver } from "../../src";
+
+const app = new App({
+  autoSynth: false,
+});
+const stack = new Stack(app, "stack");
+
+const api = new appsync.GraphqlApi(stack, "API", {
+  name: "api",
+});
+
+new AppsyncResolver(
+  stack,
+  "getPost",
+  {
+    api,
+    typeName: "Query",
+    fieldName: "getPost",
+  },
+  // valid - inline arrow function
+  () => null
+);
```

Comment:
Love these tests.

They do get stuck periodically though

![image](https://user-images.githubusercontent.com/289213/174610932-93f2c762-54a7-40f1-8d2f-45c7a83f8aac.png)

Reply:
Hmm that's odd, any idea why that would be? I've never seen that happen.

Reply:
I am not sure, never get an error, just spins. Running it on its own works fine, maybe a race condition?